website/components/CodeBlock/shiki/ShikiCodeBlockClient.tsx

```diff
@@ -0,0 +1,55 @@
+'use client'
+import parse from 'html-react-parser'
+
+import Image, { StaticImageData } from 'next/image'
+import { useClipboard, Button } from '@chakra-ui/react'
+
+import copyIcon from '../../../public/copyIcon.png'
+import ts from '../../../public/ts.png'
+import js from '../../../public/js.png'
+import bash from '../../../public/bash.png'
```

Comment:
Can we use the icons from `react-icons`? In this case, `SiTypescript`, `SiJavascript`, `SiGnuBash` and `MdOutlineContentCopy`.